.env

```diff
@@ -1 +1,2 @@
-PORT=4200
```

Comment:
Remove this file. I'm not sure where it came from. There's the sample.env, which is actually checked in, and then {environment}.env files that are git ignored.